Glossary project (M to Rancher E) (#1332)
rancher/rancher-docs: shared-files/_glossary.md

    </dd>
    <dt>
        Project Resource Quotas
    </dt>
    <dd>
        Similar to how namespaces divide cluster resources among multiple users, a project resource quota limits the cluster resources that a project (and its namespaces) can consume.
    </dd>
    <p>
        <b>Synonyms:</b> <i>Resource quotas</i>
    </p>
    <dt>
        Pod Security Admission (PSA)
    </dt>
    <p>
        <b>Versions:</b> Rancher v2.7.2 and later
    </p>
    <dd>
        Pod Security Admission (PSA) is a built-in Kubernetes controller that enforces the Pod Security Standards (PSS) on pods running in a namespace. PSA places requirements on a pod's security context and other related fields and categorizes pods into three levels based on their security requirements: Privileged, Baseline, and Restricted.
    </dd>
    <dt>
        Pod Security Admission (PSA) Config Template
    </dt>
    <p>
        <b>Versions:</b> Rancher v2.7.2 and later
    </p>
    <dd>

>         If you have administrator privileges, you can customize security restrictions and permissions by creating additional PSA templates, or by editing existing templates. Pod Security admission (PSA) configuration templates are a Rancher custom-defined resource (CRD), available in Rancher v2.7.2 and above. The templates provide pre-defined security configurations that you can apply to a cluster:

Reviewer: ```suggestion
        A Rancher custom-defined resource (CRD) that provide pre-defined security configurations that you can apply to a cluster. If you have administrator privileges, you can customize security restrictions and permissions by creating additional templates, or by editing existing templates. 
```

>         <ul>
>         <li>rancher-privileged: The most permissive configuration. It doesn't restrict the behavior of any pods. This allows for known privilege escalations. This policy has no exemptions.</li>
>         <li>rancher-restricted: A heavily restricted configuration that follows current best practices for hardening pods. You must make namespace-level exemptions for Rancher components.</li>
>         </ul>

Reviewer: ```suggestion
```

I feel like we don't need to list the levels here, just for brevity.

    </dd>
    <dt>
        Pod Security Policies (PSP)
    </dt>
    <dd>

>         Pod Security Policies (PSPs) are objects that control security-sensitive aspects of the pod specification (such as root privileges). If a pod does not meet the conditions specified in the PSP, Kubernetes will not allow it to start. PodSecurityPolicy was deprecated in Kubernetes v1.21, and removed from Kubernetes in v1.25.

Reviewer: ```suggestion
        Kubernetes objects that control security-sensitive aspects of the pod specification, such as root privileges. If a pod does not meet the conditions specified in the PSP, Kubernetes will not allow it to start. PodSecurityPolicy was deprecated in Kubernetes v1.21, and removed from Kubernetes in v1.25.
```

    <p>
        <b>Related terms:</b> <i>K3s, RKE2</i>
    </p>
    <dt>
        Rancher Chart
    </dt>
    <p>
        <b>Versions:</b> Rancher v2.6 and later
    </p>
    <dd>

>         Rancher charts differ slightly from Helm charts in their directory structures. They are native Helm charts with two files that enhance user experience: app-readme.md and questions.yaml. Rancher charts add simplified chart descriptions and configuration forms to make application deployment easy, allowing Rancher users to understand how to launch an application without having to read through the entire list of Helm variables.

Reviewer: ```suggestion
        Native Helm charts with two files that enhance the Rancher user experience: `app-readme.md` and `questions.yaml`. Rancher charts add simplified chart descriptions and configuration forms to make application deployment easier, allowing users to understand how to launch an application without having to read through the entire list of Helm variables.
```

    </dd>
    <dt>
        Rancher CLI
    </dt>
    <dd>

>         The Rancher CLI (Command Line Interface) is a unified tool that you can use to interact with Rancher. With this tool, you can operate Rancher using a command line rather than the GUI.

Reviewer: ```suggestion
        A unified tool for interacting with Rancher through a command line rather than the GUI.
```

    </dd>
    <dt>
        Rancher Enterprise
    </dt>
    <p>
        <b>Versions:</b> Rancher v2.7 and later
    </p>
    <dd>

>         Rancher Enterprise, or Rancher Prime, is a new edition of the commercial enterprise offering built on the same source code. Installation assets are hosted on a trusted registry owned and managed by Rancher, with additional value coming from security assurances, extended lifecycles, access to focused architectures, and Kubernetes advisories.

Reviewer: ```suggestion
        Also known as Rancher Prime; a new edition of the commercial enterprise offering built on the same source code. Installation assets are hosted on a trusted registry owned and managed by Rancher, with additional value coming from security assurances, extended lifecycles, access to focused architectures, and Kubernetes advisories.
```
